lib/rules/require-dictionary-words.js

```diff
@@ -0,0 +1,331 @@
+/**
+ * Only allow words defined in a dictionary or by you.
+ *
+ * Type: `Boolean` or `Object`
+ *
+ * Values:
```

Comment:
This doc should probably describe how to install third party word lists, correct?